lib/foreman_ansible/register.rb

```diff
@@ -0,0 +1,52 @@
+# rubocop:disable BlockLength
+Foreman::Plugin.register :foreman_ansible do
+  requires_foreman '>= 1.16'
+
+  security_block :foreman_ansible do
+    permission :play_roles_on_host,
+               { :hosts => [:play_roles, :multiple_play_roles],
+                 :'api/v2/hosts' => [:play_roles,
+                                     :multiple_play_roles] },
+               :resource_type => 'Host'
+    permission :play_roles_on_hostgroup,
```

Comment:
do we need separate permission for this if at the end it's just playing roles on separate hosts? I think just "play_roles" should be enough. Or we could just rely on `create_job_invocation` which is what REX already does. I see this is not the new code so it's fair to keep it out of this PR.